### DISCUSSION TITLE
Super Super Slow 4k Encoding

### DISCUSSION THREAD

**Discussion by carnahst** [jlesage/docker-handbrake#283] [Q&A]
I have installed this on my Synology NAS (DSM 7.2-64570 Update 3). The Synology box has 32GB of ram and no hardware encoding DS1621+.

Encoding HD files run in the 200 to 400 fps range.

Encoding 4K files run in the .01 or lower fps range and the whole Synology box becomes unresponsive.

I have no issues running the same 4K files through the desktop version of handbrake on Windows.

I'm new to docker, so any tips would be appreciated.

Thanks!

**Comment by jlesage** [comment]
Are you using a default preset to perform the 4K encoding ?  Which one ?

> I have no issues running the same 4K files through the desktop version of handbrake on Windows.

You mean in a VM that runs on the same NAS ?

**Reply by carnahst** [reply]
Hi,

I've been struggling with this, but I think it's resolved. I'm running this on a Synology box. It appears that something odd was happening with the box itself. The VM wasn't allocating memory properly. It's a new box, so I reset it and reinstalled. I'm now getting around 20 fps for 4k encodes.

Thanks for taking time to respond.

**Reply by carnahst** [reply]
In response to your questions:

I was having issues running this in container manager on the Synology, using the built in 4k matroska h264 preset.

**Reply by jlesage** [reply]
Thanks for the update !
